pom.xml

```diff
@@ -52,7 +51,22 @@
         <artifactoryResolveReleaseRepo>virtual-release</artifactoryResolveReleaseRepo>
 
         <environment-reader-library.version>1.2.0-rc1</environment-reader-library.version>
+        
+		<spring-boot-dependencies.version>2.1.15.RELEASE</spring-boot-dependencies.version>
+		<spring-boot-maven-plugin.version>2.1.15.RELEASE</spring-boot-maven-plugin.version>
     </properties>
+    
+    <dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>org.springframework.boot</groupId>
+				<artifactId>spring-boot-dependencies</artifactId>
+				<version>${spring-boot-dependencies.version}</version>
+				<type>pom</type>
+				<scope>import</scope>
+			</dependency>
+		</dependencies>
+	</dependencyManagement>
```

Comment:
The indentation should be 4 characters per tab, this looks like it is something like 10 characters? Please can this be re-formatted to be 4 spaces per tab.

Reply:
This should be correct now